src/main/java/controller/UserListController.java

```diff
@@ -21,7 +19,6 @@ public View doGet(HttpRequest httpRequest, HttpResponse httpResponse) {
     }
 
     private boolean isLogined(HttpRequest httpRequest) {
-        return Arrays.stream(httpRequest.getCookies())
-                .anyMatch(cookie -> cookie.contains("SESSIONID="));
+        return "true".equals(httpRequest.getCookie().get("logined"));
```

Comment:
[Boolean.parseBoolean()](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#parseBoolean(java.lang.String)) 메소드를 사용해도 좋을 것 같아요.

```suggestion
        return Boolean.parseBoolean(httpRequest.getCookie().get("logined"));
```